roles/cds/tasks/main.yml

```diff
@@ -1,37 +1,92 @@
-- name: Install cds package
-  ansible.builtin.apt:
-    pkg: icpc-host-cds
-    state: present
+---
+- name: Create cds user
+  ansible.builtin.user:
+    name: cds
+    home: /srv/cds
+    shell: "/usr/bin/bash"
 
-- name: Check if cds is installed
+- name: Create CDS directories
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: directory
+    owner: cds
+    group: cds
+    mode: '0755'
+  with_items:
+    - /srv/cds
+    - /var/lib/cds
+
+- name: Get the CDS release
+  ansible.builtin.uri:
+    url: https://api.github.com/repos/icpctools/icpctools/releases?per_page=1
+    method: GET
+    return_content: true
+    status_code: 200
+    body_format: json
+  register: latest_cds_release_array
+  check_mode: false
+
+- name: Set CDS latest version
```

Comment:
maybe we should put this whole github scraping section in a block, when the version is latest do the scraping if its set in a variable use that. We tend to use the latest at our contests but if/when that is broken pinning temporary in an easy way will make reuse easier. If you agree we can create an issue for this and I can do this (so it shouldn't be blocking for this PR)

Reply:
Yeah. That seems reasonable to me.